app/src/main/java/pl/futurecollars/invoicing/controller/tax/TaxCalculator.java

```diff
@@ -0,0 +1,19 @@
+package pl.futurecollars.invoicing.controller.tax;
```

Comment:
Jeśli dodajesz nowy pakiet na tax, to invoice też pprzenieś do odpowiedniego pakietu